final-project/styles/main.scss

```diff
@@ -2,12 +2,1164 @@
+    top: 110%;
+    width: 100%;
+
+    &__bar {
+      display: flex;
+      justify-content: center;
+    }
+
+    &__item {
+      background-color: #D1D6E3;
+      width: 10px;
+      height: 10px;
+      margin: 0 5px;
+      border-radius: 50%;
+
+      &--active {
+        background-color: #5283FF;
+      }
+    }
+  }
+
+}
+
+.customers {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  background-color: #EAF0FF;
+  height: Rem(480px);
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    
+    &--left {
+      align-items: flex-end;
+    }
+
+    &--right {
+      align-items: flex-start;
+    }
+
+    &__container {
+      width: 75%;
+    }
+  }
+
+  &__title {
+    color: $primary-color;
+    font-size: Rem(36px);
+    margin: Rem(40px) 0 Rem(25px);
+    font-weight: 100;
+  }
+
+  &__description {
+    width: 70%;
+    font-size: Rem(12px);
+  }
+
+  &__carousel {
+    position: relative;
+    width: 60%;
+    height: 50%;
+  }
+
+  &__card {
+    background-color: white;
+    width: 80%;
+    height: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 10% 10%;
+    font-size: Rem(13px);
+  }
+
+  &__image {
+    width: Rem(52px);
+    height: Rem(52px);
+    background-color: #4C85FF;
+    border-radius: 50%;
+    position: absolute;
+    left: calc(50% - 26px);
+    bottom: Rem(.2px);
+  }
+}
+
+.faq {
+  padding: Rem(175px) Rem(150px) Rem(100px);
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    margin-top: Rem(20px);
+    font-weight: 300;
+    color: $primary-color;
+  }
+
+  &__container {
+    margin-top: Rem(50px);
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__list {
+    width: 100%;
+
+    &__item {
+      cursor: pointer;
+      padding: Rem(40px) 4%;
+      display: flex;
+      flex-direction: column;
+      border-bottom: #D8E3FE Rem(2px) solid;
+
+      &__title {
+        display: flex;
+        align-items: center;
+      }
+
+      &__question {
+        margin-left: Rem(30px);
+
+        &--active {
+          color: #495274;
+          font-weight: bold;
+        }
+      }
+
+      &__answer {
+        display: none;
+        margin: Rem(20px) 0 0 10%;
+
+        &--active {
+          display: flex;
+        }
+
+        &__text {
+          width: 60%;
+        }
+
+        &__btn {
+          cursor: pointer;
+          font-size: Rem(13px);
+          color: #7382A3;
+          background-color: #E8ECF4;
+          width: Rem(195px);
+          height: Rem(43px);
+        }
+      }
+      
+    }
+  }
+}
+
+.bg-clouds-blue {
+  overflow: hidden;
+  
+  &__image {
+    width: 100%;
+  }
+}
+
+.plan-life {
+  display: none;
+}
+
+.newsletter {
+  height: Rem(110px);
+  padding: Rem(102px) 0;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  border-bottom: Rem(1px) solid $text-primary;
```

Comment:
It is great you are using grid! In this section, on mobile, the input and the button looks vertical stretched and horizontal compacted, maybe here is a good place to put a media query and change the grid to be stacked :)